Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -367,9 +367,9 @@ PhysicalParticleContainer::AddPlasmaFromFile(const std::string s_f,
         std::shared_ptr<amrex::ParticleReal> ptr_uz = ps.second["momentum"]["z"].loadChunk<amrex::ParticleReal>();
         double const momentum_unit_z = ps.second["momentum"]["z"].unitSI();
 #   else
-        amrex::Abort("AddPlasmaFromFile is only implemented for 2D and 3D\n")
+        amrex::Abort("AddPlasmaFromFile is only implemented for 2D and 3D\n");
 #   endif
-#   if (defined WARPX_DIM_3D)
+#   if (defined WARPX_DIM_XZ)
```

Comment:
Just to double check, is this supposed to be XZ and not 3D?

Reply:
```suggestion
#   if (defined WARPX_DIM_3D)
```

Reply:
Oops, absolutely, thanks for the catch.